app/javascript/packages/spinner-button/spinner-button-element.spec.ts

```diff
@@ -1,23 +1,29 @@
 import sinon from 'sinon';
-import userEvent from '@testing-library/user-event';
+import baseUserEvent from '@testing-library/user-event';
 import { getByRole, fireEvent, screen } from '@testing-library/dom';
 import './spinner-button-element';
 import type { SpinnerButtonElement } from './spinner-button-element';
 
 describe('SpinnerButtonElement', () => {
-  let clock;
+  let clock: sinon.SinonFakeTimers;
+  const userEvent = baseUserEvent.setup({ advanceTimers: (ms: number) => clock.tick(ms) });
 
   const longWaitDurationMs = 1000;
 
   interface WrapperOptions {
     actionMessage?: string;
 
     tagName?: string;
+
+    spinOnClick?: boolean;
   }
 
-  function createWrapper({ actionMessage, tagName = 'a' }: WrapperOptions = {}) {
+  function createWrapper({ actionMessage, tagName = 'a', spinOnClick }: WrapperOptions = {}) {
     document.body.innerHTML = `
-      <lg-spinner-button data-long-wait-duration-ms="${longWaitDurationMs}">
+      <lg-spinner-button
+        data-long-wait-duration-ms="${longWaitDurationMs}"
+        ${spinOnClick === undefined ? '' : `spin-on-click="${spinOnClick}"`}
```

Comment:
Should probably make these consistent. Recently I'm less inclined to do everything as `data-` attributes, since it's more verbose, and `dataset` keys/values aren't usually the correct name or value type in their raw form, so trying to leverage it directly is pretty fraught.

Reply:
Updated in [d64bff8](https://github.com/18F/identity-idp/pull/6287/commits/d64bff8b81016a0070c6930503e24812055ff2d7).